Resources/Textures/Constructible/Power/VendingMachines/boozeomat.rsi/meta.json

```diff
@@ -1 +1,49 @@
-{"version": 1, "size": {"x": 32, "y": 32}, "states": [{"name": "broken", "directions": 1, "delays": [[1.0]]}, {"name": "deny", "directions": 1, "delays": [[0.2, 0.2, 0.2, 0.2, 0.2, 0.2, 0.2, 0.2]]}, {"name": "normal", "directions": 1, "delays": [[0.2, 0.2, 0.2, 0.2, 0.2, 0.2, 0.2, 0.2]]}, {"name": "off", "directions": 1, "delays": [[1.0]]}]}
\ No newline at end of file
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
```

Comment:
Seems like this and other RSIs in your PR are missing license/copyright information. Could you add this, please? Thank you